### ISSUE TITLE
Toggle between different score types in the score report

### ISSUE THREAD

**Issue by jyeatman** [yeatmanlab/roar#60]
Right now the score table toggles between colored dots and percentiles. Under the hood raw scores and standard scores exist. My first intuition is to make this a 4 way toggle to go between these 4 types of scores: dots / percentiles / standard scores / raw scores.

But putting this as a comment for discussion and tagging @ctflores

**Comment by ctflores** [comment]
Agree! Not knowing the engineering that has to go on behind the scenes, I also agree with the prioritization of (1) dots (2) percentiles (3) standardized scores (4) raw

**Comment by jyeatman** [comment]
@kellyel it sounds like there is agreement on making this a four way toggle assuming that is easy to do. In other words, if the scores are there, allowing the user to toggle through them would be a very useful feature 

Tagging @richford for thoughts.

**Comment by kellyel** [comment]
I agree that giving more info to the user would be helpful here. I'm going to have to look into the implementation of this, as I believe allowing all 4 fields to be swapped out might be a bit more involved than just the first 2. Regardless, I know it will be possible.

**Comment by jyeatman** [comment]
@kellyel why don't you let us know what options are "easy" to implement. Because I think this is very useful if it's easy but not a high priority if it is at the expense of other critical issues

**Comment by richford** [comment]
This is currently implemented via a dropdown menu.
